src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,237 @@
+---
+title: CI/CD for API Mesh
+description: Learn how to set up continuous integration and continuous delivery/deployment for API Mesh .
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# CI/CD for API Mesh
+
+API Mesh for Adobe Developer App Builder now provides CI/CD through [Github actions](https://docs.github.com/en/actions). This allows developers to build, test, and deploy meshes faster and more reliably.
```

Comment:
We should include a note that explains that API Mesh and App Builder now leverage the same tools to enable CI/CD through Github Actions, so the experience should be consistent for both products.